client/a8c-for-agencies/sections/marketplace/pressable-overview/plan-selection/details.tsx

```diff
@@ -115,6 +115,13 @@ export default function PlanSelectionDetails( {
 							) }
 						</div>
 					) }
+					{ isReferMode && (
+						<div className="pressable-overview-plan-selection__details-card-header-price">
+							<strong className="pressable-overview-plan-selection__details-card-header-comming-soon">
```

Comment:
[NITPICK]

There is a typo here:

```suggestion
							<strong className="pressable-overview-plan-selection__details-card-header-coming-soon">
```